sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -63,6 +63,10 @@ singleTableIdentifier
     : tableIdentifier EOF
     ;
 
+singleMultiPartIdentifier
```

Comment:
This is a top-level parser entry used in `ParserInterface`. I don't think we need it now for catalog identifier.

Reply:
True, only my test case uses it to parse a table name into a sequence. I will remove it.

Reply:
Won't we need this eventually for parsing names passed to `saveAsTable`? Why not add it now?

Reply:
When I start to to convert SELECT, INSERT, and DROP code path to support multi-catalog, this parse function is needed, e.g,
```
  override def visitTable(ctx: TableContext): LogicalPlan = withOrigin(ctx) {
    UnresolvedIdentifier(visitMultiPartIdentifier(ctx.multiPartIdentifier))
  }

 override def visitTableName(ctx: TableNameContext): LogicalPlan = withOrigin(ctx) {
    val tableId = visitMultiPartIdentifier(ctx.multiPartIdentifier())
    val table = mayApplyAliasPlan(ctx.tableAlias, UnresolvedIdentifier(tableId))
    table.optionalMap(ctx.sample)(withSample)
  }
```